File: 20221013/모범답안/20221013_05/police.py
def solution(n, k, theives):
    """
    :param n: int
    :param k: int
    :param theives: list
    :return: int
    """
    police, answer = 0, 0
    theives.sort(reverse=True)
    
    for i in range(k):
        if theives[i] > police:
            answer += 1
            police += n - theives[i]

    return answer

File: 20221013/모범답안/20221013_05/police.js
/**
 * @param {int} n
 * @param {int} k
 * @param {array} thieves
 * @return {int}
 */
function solution(n, k, theives){
    let police = 0
    let answer = 0
    theives.sort(function(a,b){
        return b - a;
    });
    
    for(let i = 0; i < k; i++){
      if (police >= theives[i]) break
      police += n - theives[i]
      answer++
    }

    return answer
}
